src/libs/Navigation/drawerGoBack/index.js

```diff
@@ -0,0 +1,13 @@
+import Navigation from '../Navigation';
+
+/**
+ * @private
+ * @param {string} backRoute
+ */
+export default function drawerGoBack(backRoute) {
```

Comment:
@narefyev91 @techievivek what do you think of putting this method in `Navigation.js`, so that when it's imported and called it is called as `Navigation.drawerGoBack`.

The keyboard hiding part can be platform specific with `isKeyboardVisible` check on Android and for others a noop function.